src/main/resources/logback.xml

```diff
@@ -10,8 +10,9 @@ DEBUG
 INFO
 WARN
 ERROR-->
-	<logger name="org.usfirst.frc.team1758.robot.commands.CenterRobotTarget" level="TRACE"/>
-
+	<logger name="org.usfirst.frc.team1758.robot.commands.OrientTarget" level="TRACE"/>
+	<logger name="org.usfirst.frc.team1758.robot.commands.ApproachPeg" level="TRACE"/>
+	<logger name="org.usfirst.frc.team1758.robot.commands.ToggleGear" level="TRACE"/>
```

Comment:
Lets remove these when we get to Competition as this is a taxing process on the Robot. It is nice for debug but not the best for compotition. May want to remove these lines (We will still get Warn and Error if they are removed.)